### PR TITLE
Enhanced UI of mark exit page according to design file

### LINKED ISSUE #4: UI enhancement of Mark Exit page
Sir, the UI of Mark Exit page in the hostelite page is not up to mark. The input text areas are not correctly aligned and the overall UI looks messed up. @RitwikSingh28 Sir kindly assign this issue to me so that I can improve it as per the design file.
WhatsApp Image 2023-10-12 at 00 28 24_2290b8f5

### PR DESCRIPTION
**Pull request by ShubhamTiwari55**
@RitwikSingh28 Sir the issue #4 has been resolved now. I have added blur effects to background image so that the page becomes more readable and adjusted elements by giving them proper padding and margin. Also I have added the hostelite image in the bottom as per the design file.
WhatsApp Image 2023-10-13 at 11 40 57_f9ddcf71

### CODE REVIEW

File: lib/student_screens/mark_exit.dart [hunk 1]
```diff
@@ -102,91 +105,103 @@ class _MarkingExitState extends State<MarkingExit> {
...
       ),
-      body: Padding(
-        padding: const EdgeInsets.all(20),
-        child: Container(
-          constraints: BoxConstraints.expand(),
-          decoration: BoxDecoration(
+      body: Stack(
+        children: [
+          Container(
+            constraints: BoxConstraints.expand(),
```
Comment (RitwikSingh28) [inline lib/student_screens/mark_exit.dart:111]: The background image should have some margin from the left-side of the screen. 

@ShubhamTiwari55 Kindly add a left-margin

**Review by RitwikSingh28 — CHANGES_REQUESTED** [review]
Please make the change specified in the comment.

**Comment by ShubhamTiwari55** [comment]
@RitwikSingh28 Sir the required change i.e adding left margin has been done. Kindly review the updated PR.

WhatsApp Image 2023-10-14 at 13 03 13_08123499

**Review by RitwikSingh28 — APPROVED** [review]
Great work! :partying_face: 

Merging into `main` branch

**Pull request merged**